package.json

```diff
@@ -27,20 +28,38 @@
     "qrcode-reader": "^1.0.4"
   },
   "devDependencies": {
+    "@commitlint/cli": "^8.2.0",
+    "@commitlint/config-angular": "^8.2.0",
     "eslint": "^6.5.1",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.4.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.1.1",
     "execa": "^2.0.5",
     "http-server": "^0.11.1",
+    "husky": "^3.0.8",
     "jest": "^24.9.0",
+    "lint-staged": "^9.4.2",
     "pkg": "^4.4.0",
     "prettier": "^1.18.2",
     "semantic-release": "^15.13.24"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/victorperin/qr-scanner-cli.git"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "eslint --fix",
+      "prettier --write",
+      "git add",
+      "yarn test:diff"
```

Comment:
can you change here to `npm run test:diff`? Just because some people don't use yarn (myself included).